workspaces/sample-workspace/client-extensions/sample-theme-spritemap-3/client-extension.yaml

```diff
@@ -0,0 +1,9 @@
+assemble:
+    - from: build
+      include: "spritemap.svg"
+      into: static
+sample-theme-spritemap-3:
+    extendClay: true
```

Comment:
Is  this going to be used at deploy or runtime for anything? :thinking: 

Because if it is only used to build the CE I don't think this file would be the proper place. We may need to move it back to `liferay.json` again... :thinking: 

This is bc all these properties end up in DXP's DB and will show in the CE UI, thus they have to make sense to the end user/sysadmin (as opposed to the developer).

Reply:
> We may need to move it back to liferay.json again... 🤔

😭 

So having liferay.json might be unavoidable then for a CET that extends from the clay spritemap. Unless we so something like check the dependency in the package.json or something.

Reply:
I wouldn't be so sad about the `liferay.json` file. If you give it a thought, when you integrate @angular/cli, create-remote-app, tsc, ... you need their config files too :shrug: 

As long as most of the configuration values are inferrable from the project environment, we can omit the `liferay.json` file except for very special tweaks. For example, checking the dependency in the `package.json` as you suggest would be good. We just put that check in `Project.ts` and infer that value.

Then we can add some feature to `@liferay/cli` to dump the effective configuration it is using, so that people know what it is inferring from the environment.